src-vue/src/views/RepairView.vue

```diff
@@ -1,30 +1,30 @@
 <template>
     <div class="fc-container">
         <el-scrollbar>
-            <el-alert title="Info" type="info" :closable="false" show-icon>
-                This window contains various functionality to repair common issues with Northstar and FlightCore.
+            <el-alert title="Information" type="info" :closable="false" show-icon>
```

Comment:
Should that line also be translated?

Reply:
Which line are you talking about?

Reply:
You're right, addressed in dc468a921b84ab259cfa21b12116e8cb71c7fd5b